express-notes-api/index.js

```diff
@@ -0,0 +1,128 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+const dataJson = require('./data.json');
+
+app.get('/api/notes', (req, res) => {
+  const notesArr = [];
+  for (const properties in dataJson.notes) {
+    notesArr.push(dataJson.notes[properties]);
+  }
+  res.send(notesArr);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  const id = Number(req.params.id);
+  if (id < 1 || isNaN(id)) {
+    const errMsg = {
+      error: 'id must be a positive integer'
+    };
+    res.status(400).json(errMsg);
```

Comment:
An `id` that is a non-integer number should also result in a `400 Bad Request` response.